src/test/java/subway/StationAcceptanceTest.java

```diff
@@ -14,54 +13,75 @@
 import java.util.Map;
 
 import static org.assertj.core.api.Assertions.assertThat;
+import static subway.StationSteps.*;
 
 @DisplayName("지하철역 관련 기능")
-@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.DEFINED_PORT)
-public class StationAcceptanceTest {
+public class StationAcceptanceTest extends ApiTest {
     /**
      * When 지하철역을 생성하면
      * Then 지하철역이 생성된다
      * Then 지하철역 목록 조회 시 생성한 역을 찾을 수 있다
      */
-    @DisplayName("지하철역을 생성한다.")
+    @DisplayName("지하철역을 생성한다")
     @Test
     void createStation() {
         // when
-        Map<String, String> params = new HashMap<>();
-        params.put("name", "강남역");
-
-        ExtractableResponse<Response> response =
-                RestAssured.given().log().all()
-                        .body(params)
-                        .contentType(MediaType.APPLICATION_JSON_VALUE)
-                        .when().post("/stations")
-                        .then().log().all()
-                        .extract();
+        String stationName = "강남역";
+        StationRequest request = 지하철역_생성_요청(stationName);
+        ExtractableResponse<Response> createStationResponse = 지하철역_생성(request);
 
         // then
-        assertThat(response.statusCode()).isEqualTo(HttpStatus.CREATED.value());
+        assertThat(createStationResponse.statusCode()).isEqualTo(HttpStatus.CREATED.value());
 
         // then
-        List<String> stationNames =
-                RestAssured.given().log().all()
-                        .when().get("/stations")
-                        .then().log().all()
-                        .extract().jsonPath().getList("name", String.class);
-        assertThat(stationNames).containsAnyOf("강남역");
+        ExtractableResponse<Response> showStationsResponse = 지하철역_조회();
+        List<String> stationNames = showStationsResponse.jsonPath().getList("name", String.class);
+        assertThat(stationNames).containsAnyOf(stationName);
     }
 
     /**
      * Given 2개의 지하철역을 생성하고
      * When 지하철역 목록을 조회하면
      * Then 2개의 지하철역을 응답 받는다
      */
-    // TODO: 지하철역 목록 조회 인수 테스트 메서드 생성
+    @DisplayName("지하철역 목록을 조회한다")
+    @Test
+    void showStations() {
+        // given: 2개의 지하철역 생성
+        String stationName1 = "강남역";
+        지하철역_생성(지하철역_생성_요청(stationName1));
+
+        String stationName2 = "역삼역";
+        지하철역_생성(지하철역_생성_요청(stationName2));
+
+        // when: 지하철역 목록 조회
+        ExtractableResponse<Response> response = 지하철역_조회();
+
+        // then: 생성되어 있는 2개의 지하철역 응답
+        assertThat(response.statusCode()).isEqualTo(HttpStatus.OK.value());
+        List<String> stationNames = response.jsonPath().getList("name", String.class);
+        assertThat(stationNames).contains(stationName1, stationName2);
```

Comment:
```suggestion
        assertThat(stationNames).containsExactly(stationName1, stationName2);
```

`containsExactly`를 사용해 볼까요?

Reply:
`containsExactly`를 사용하는 경우 `지하철역_조회` 기능의 정렬 순서까지 테스트케이스에서 검사를 해줄 것 같은데요.

정의한 케이스에서는 정렬 순서까지는 정하지 않았기에 저는 `contains`을 활용하게 되었습니다!

혹시 플람께서 `containsExactly` 사용을 제안주신 의미를 알려주실 수 있을까요??

Reply:
정확한 순서와 중복까지 검사하기 위해서는 `containsExactly`이 좋다고 생각합니다, 만약 정의한 케이스에서 정렬 순서에 대해 정하지 않으셨다면, `containsOnly`, `contains`를 사용하시는 것이 좋긴 하겠네요 :) 

저는 일단 순서까지 정확하게 검증하는 것을 선호하여 `containsExactly`를 사용합니다.